src/app/components/navbar/index.jsx

```diff
@@ -1,13 +1,20 @@
 import { connect } from "react-redux";
-import { getActive, getGroupsLoading, getEnableNewSignIn } from "../../config/selectors";
+import { getActive, getPreviewLanguage } from "../../config/selectors";
+import { setPreviewLanguage } from "../../config/actions";
 import NavBar from "./NavBar";
 
 const mapStateToProps = state => ({
     user: state.user,
     rootPath: state.state.rootPath,
     workingOn: getActive(state.state),
     config: state.state.config,
-    isNewSignIn: getEnableNewSignIn(state.state),
+    previewLanguage: getPreviewLanguage(state.state),
```

Comment:
Bit confused on the state syntax here. 

What exactly is state versus state.state?

Why is user state.user and not state.state.user?

Reply:
ok, so it is because someone designed this this way. it call comes form the store.js and this line:
https://github.com/ONSdigital/florence/blob/develop/src/app/config/store.js#L19
you can see users are in separate reducer and this is why coming under their name. 
Ideally I would like to change that just seems very tricky before refactor